pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -184,7 +184,7 @@ func cleanupFinishedJobs(cj *batchv1.CronJob, js []batchv1.Job, jc jobControlInt
 	// Update the CronJob, in case jobs were removed from the list.
 	if _, err := cjc.UpdateStatus(cj); err != nil {
 		nameForLog := fmt.Sprintf("%s/%s", cj.Namespace, cj.Name)
```

Comment:
`nameForLog := fmt.Sprintf("%s/%s", cj.Namespace, cj.Name)`

Per https://github.com/kubernetes/klog/blob/master/klog.go#L1583-L1595, klog.KObj  is used on Pod(namespace and name)

Not sure if `nameForLog` is a better way.

Reply:
Will fix this.